website/content/en/docs/faqs/_index.md

```diff
@@ -238,3 +238,21 @@ allow configMaps larger than `~1MB`. However, from [OLM](https://github.com/oper
 
 The change to allow bigger bundles from [OLM](https://github.com/operator-framework/operator-lifecycle-manager) version `v0.19.0` only impacts the full bundle size amount. 
 Any single manifest within the bundle such as the CRD will still make the bundle uninstallable if it exceeds the default file size limit on clusters (`~1MB`).
+
+### The size of my Operator bundle is too big. How can I keep the CRDs smaller?
+
+We can design Operator solutions that have many APIs(CRDs). In this way, you might want to review the design of your Operator solution to ensure that it is not hurting concepts such as; encapsulation, the single responsibility principle, and cohesion. Also, you might need to find some alternative options to achieve the same goal in another way. To better understand the APIs, see [Groups and Versions and Kinds oh my!][kb-apis].
+
+Following are some general guidelines that we hope help you out:
+
+- Do all definitions must be in a single API(CRD/CR), or could it make sense to be split in a better API design?
```

Comment:
how does splitting into more CRDs help reduce the size of the bundle? If anything i'd expect turning 1 CRD into 2 CRDs would make the bundle bigger (overhead per CRD)

Reply:
it is over `How can I keep the CRDs smaller?`

Reply:
how does keeping the CRDs smaller help with the overall CSV size constraint?

Reply:
to be clear, the 1mb limit applies to any single yaml in the bundle, but generally the only one that ever risks exceeding 1mb is the CSV, right?

have we ever seen a single CRD yaml exceed 1mb?

Reply:
I just checked cases where the total size was bigger than 1MB so far. Not the files.